.travis.yml

```diff
@@ -10,10 +10,17 @@ addons:
     - liblapack-dev
     - python-numpy
     - swig
+before_script:
+  - wget https://repo.continuum.io/miniconda/Miniconda3-4.3.14-Linux-x86_64.sh
```

Comment:
Do you really need to install Miniconda for this?  Can't you just add `python3-numpy` to the Ubuntu dependencies and then configure with `PYTHON=python3` below?